client/reader/stream/index.jsx

```diff
@@ -39,7 +39,7 @@ import CombinedCard from 'blocks/reader-combined-card';
 import fluxPostAdapter from 'lib/reader-post-flux-adapter';
 import config from 'config';
 import { keysAreEqual } from 'lib/feed-stream-store/post-key';
-import { isDiscoverBlog, isDiscoverFeed } from 'reader/discover/helper';
+import { combineCards, injectRecommendations, RECS_PER_BLOCK } from './utils';
```

Comment:
I think if we're using `RECS_PER_BLOCK` across files, we should probably pop it into shared config instead.